src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Services/QueryStringParameterBuilder.cs

```diff
@@ -9,7 +9,16 @@ public static class QueryStringParameterBuilder
     {
         var parameters = new Dictionary<string, string[]>();
         if (!string.IsNullOrWhiteSpace(request.Keyword)) parameters.Add("keyword", new[] { request.Keyword });
-        if (request.FromDate != null) parameters.Add("fromDate", new[] { request.FromDate });
+        if (request.FromDate == null || request.FromDate == DateTime.Today.ToString("yyyy-MM-dd"))
+        {
+            var fromDate = DateTime.Now.ToUniversalTime();
```

Comment:
Did you mean to use DateTime.UtcNow here?

Reply:
changed